lib/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "mailjet-rs"
-version = "0.1.2"
+version = "1.0.0"
```

Comment:
Though we wanted to keep the bump as a patch and hold on with Tokio 1.x?

Reply:
Did I misunderstand `have Tokio 1.x support for Mailjet-RS at 1.x` ?

Reply:
I would like to support tokio ^1 with my pull request

Reply:
What I understand is:
0.X.X supports 0.2.X tokio
1.X.X supports 1.X.X tokio
Then mailjet-rs can still be maintained for both version of tokio

Reply:
Oh, we can also use `feature` keyword to allow 0.2.x tokio WDYT ? If so, any name recommandation for the name ? And which version ?

Reply:
For a v1.0.0 I would like to introduce some changes to the API to make it more convenient and have this major release with support for the Send API v3.1.

We would introduce Tokio v1.0.0 as well in that case. Does that sounds good?

Reply:
@TimeEngineer based on our [latest discussion](https://github.com/EstebanBorai/mailjet-rs/pull/7#issuecomment-890591568) let's go with  `0.2.0` as the version for this release with Tokio ^1